config/core/test-configs.yaml

```diff
@@ -2281,6 +2281,18 @@ test_configs:
   - device_type: qcom-qdf2400
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ima
+      - ltp-ipc
+      - ltp-mm
+      - ltp-pty
+      - ltp-timers
```

Comment:
It worked but with a few failures: https://staging.kernelci.org/test/plan/id/618aa40b271f3b3ae7a6c43f/